docs/api-docs/storefront/graphql/graphql-products.md

```diff
@@ -0,0 +1,875 @@
+# Products with the GraphQL Storefront API
+
+BigCommerce's GraphQL Storefront API lets merchants on headless storefronts retrieve [products](https://support.bigcommerce.com/s/article/Product-Identifiers) powered by results from our back-end search engine. These built-in capabilities also allow Stencil developers to customize their storefronts with variant information.
+
+The GraphQL Storefront API lets you retrieve the following product features, and more:
+- Price info in a store's transacting currency   
+- Product options associated with a product (variant and modifier options) 
+- Product metafields that have storefront access
+
+You can access these features for a product if a merchant makes a product visible on storefronts.
+
+This page walks you through how to retrieve info for a product. If your product has variants, see [Variants with the GraphQL Storefront API](/api-docs/storefront/graphql/variants) for how to retrieve variant info. For full schema documentation, see the [GraphQL Storefront Playground](https://developer.bigcommerce.com/api-docs/storefront/graphql/playground).
+
+## Get a Product
+
+### Get a product with the `product` field
+
+You can query a product by using the `product` field and specifying a product identifier, for example, the product `entityId`.
+
+```graphql title="Get a product with the product field" lineNumbers
+# This query retrieves 1 product.
+
+query {
+  site {
+    product (entityId: 111) {
+      # fields on the Product object type
+    }
+  }
+}
+```
+
+&nbsp;
+
+```graphql title="Get a product with the product field" lineNumbers
+# This query retrieves 2 products.
+# This query uses aliases and fragments. For more, see https://graphql.org/learn/queries.
+
+query {
+  site {
+    product1: product(entityId: 113) {
+      ...ProductFields
+    }
+    product2: product(entityId: 115) {
+      ...ProductFields
+    }
+  }
+}
+
+fragment ProductFields on Product {
+  # fields on the Product object type 
+}
+```
+
+<!-- theme:info -->
+> #### Get product versus variant
+> The Product object can also be used to retrieve variant info. For example, if you use the identifier `variantEntityId` or `optionValueIds`, you will retrieve info for the variant overlaid on the Product object (if the variant has a different values than the product). 
+> See [Get a variant](/api-docs/storefront/graphql/variants#get-a-variant) for more info.
+
+### Get a product with the `products` field
+
+You can  query a product by using the `products` field and specifying a product identifier, for example, the product `entityID`:
```

Comment:
```suggestion
You can  query a list of products by using the `products` field and specifying a product identifier, for example, the product `entityID`:
```

Reply:
copyedited the sentence to a list of products ✔️